vulnerabilities/models.py

```diff
@@ -281,3 +317,22 @@ class VulnerabilitySeverity(models.Model):
 
     class Meta:
         unique_together = ("vulnerability", "reference", "scoring_system")
+
+
+class Advisory(models.Model):
+    """
+    An advisory directly obtained from upstream without any modifications.
+    """
+
+    date_published = models.DateField(help_text="UTC Date of publication of the advisory")
+    date_collected = models.DateField(help_text="UTC Date on which the advisory was collected")
+    source = models.CharField(
+        max_length=100,
+        help_text="Fully qualified name of the importer prefixed with the module name importing the advisory. Eg: vulnerabilities.importers.nginx.NginxDataSource",
+    )
+    date_improved = models.DateTimeField(
+        null=True, help_text="Latest date on which the advisory was improved by an improver"
+    )
+    data = models.JSONField(
```

Comment:
```suggestion
    # we use a JSON field here to avoid creating a complete relational model for data that
    # is never queried directly; instead it is only retrieved and processed as a whole by
    # an improver 
    data = models.JSONField(
```